src/triggered/Merge.ts

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement, query, TemplateResult } from 'lit-element';
+
+import { newWizardEvent } from '../foundation.js';
+import { mergeWizard } from '../wizards.js';
+
+export default class MergePlugin extends LitElement {
+  doc!: XMLDocument;
+
+  @query('#merge-plugin-input') pluginFileUI!: HTMLInputElement;
+
+  mergeDoc(event: Event): void {
+    const file =
+      (<HTMLInputElement | null>event.target)?.files?.item(0) ?? false;
+    if (file)
+      file.text().then(text => {
+        const doc = new DOMParser().parseFromString(text, 'application/xml');
+        // FIXME: Dirty hack should not be necessary!
+        document
+          .querySelector('open-scd')!
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
          .querySelector("open-scd")!
```